lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -1116,21 +1116,23 @@ class VerticalLine extends FlLine with EquatableMixin {
 /// Draws a title on the [HorizontalLine]
 class HorizontalLineLabel extends FlLineLabel with EquatableMixin {
   /// Draws a title on the [HorizontalLine], align it with [alignment] over the line,
-  /// applies [padding] for spaces, and applies [style for changing color,
+  /// applies [padding] for label padding, applies [margin] for label space andapplies [style] for changing color,
   /// size, ... of the text.
   /// Drawing text will retrieve through [labelResolver],
   /// you can override it with your custom data.
   /// /// [show] determines showing label or not.
   HorizontalLineLabel({
     EdgeInsets? padding,
+    EdgeInsets? margin,
```

Comment:
Please add margin in the larp function below

Reply:
Removed margin as we have vertical/horizontal offsets